src/webui/abstractwebapplication.cpp

```diff
@@ -244,17 +244,8 @@ QString AbstractWebApplication::generateSid()
 
 void AbstractWebApplication::translateDocument(QString& data)
 {
-    const QRegExp regex("QBT_TR\\((([^\\)]|\\)(?!QBT_TR))+)\\)QBT_TR(\\[CONTEXT=([a-zA-Z_][a-zA-Z0-9_]*)\\])?");
+    const QRegExp regex("QBT_TR\\((([^\\)]|\\)(?!QBT_TR))+)\\)QBT_TR(\\[CONTEXT=([a-zA-Z_][a-zA-Z0-9_]*)\\])");
```

Comment:
The old one looks so complex, I would write:
`QBT_TR\\((\\w+)\\)QBT_TR\\[CONTEXT=(\\w+)\\]`
but I'm not sure will work in Qt and probably not compatible with the old one.